docs/StandaloneConfiguration.md

```diff
@@ -25,13 +26,16 @@
 ```
 #### Line-by-line explanation
 ```
-"ignoreGlobList": ["**.dat", "**.js"]  <-- Repository level's list of file formats to ignore.
+"ignoreGlobList": ["**.dat", "**.js"]        <-- Repository level's list of file formats to ignore.
 
-"ignoreGlobList": ["**.css"]           <-- Author level's ignoreGlobList adds on to the Repository level's.
+"ignoreCommitList": ["12345abc", "67890def"] <-- The list of commits to ignore during analysis. For accurate
+                                           results, the commits should be provided with their full hash.
+
+"ignoreGlobList": ["**.css"]                 <-- Author level's ignoreGlobList adds on to the Repository level's.
                                            Thus, the actual ignoreGlobList for alice would contain
                                            css, dat and js.
 
-"githubId": "bob"                      <-- Only githubId is mandatory.
+"githubId": "bob"                            <-- Only githubId is mandatory.
                                            Optional information can be left out to prevent clutter.
```

Comment:
Minor nits. No need for change.
Other than arrows, text should be aligned with respective to its block.

```
<-- Only githubId is mandatory.
    Optional information can be left out to prevent clutter.
```